src/Apps/Artist/Routes/WorksForSale/ArtistWorksForSaleRoute.tsx

```diff
@@ -14,25 +20,82 @@ const ArtistWorksForSaleRoute: React.FC<ArtistWorksForSaleRouteProps> = ({
   artist,
 }) => {
   const { title, description } = artist.meta
-  const total = artist.sidebarAggregations?.counts?.total ?? 0
+  const { match } = useRouter()
 
   return (
     <>
       <Title>{title}</Title>
       <Meta name="title" content={title} />
       <Meta name="description" content={description} />
 
-      {total === 0 ? (
-        <ArtistWorksForSaleEmptyFragmentContainer artist={artist} />
-      ) : (
-        <ArtistArtworkFilterRefetchContainer
-          artist={artist}
-          aggregations={
-            artist.sidebarAggregations
-              ?.aggregations as SharedArtworkFilterContextProps["aggregations"]
-          }
+      {/* TODO: Figure out why rerenders trigger refetches here, requiring
+              the static container to freeze rendering during route transitions. */}
+      <StaticContainer shouldUpdate={!!match.elements}>
```

Comment:
FYI, can always check if a route is transitioning by looking at `match.elements`. If its undefined, route bar shows.